.gitignore

```diff
@@ -19,3 +19,8 @@ bin/manager
 
 # vim tmp files
 *.swp
+
+gomock_reflect*
+
+# side effect of asciidoctor
```

Comment:
How does that happen?

Reply:
I ran `go generate` and I maybe stopped it in the middle 😓